src/network/graphDataStruct.js

```diff
@@ -74,11 +60,253 @@ const FIELD_MAPPINGS = {
     consensusThreshold
   `,
 
-  syncInfo: `
+  SyncInfo: `
     syncBlockNum
     syncBlockTime
     chainBlockNum
   `,
+
+  Transaction: `
+    version
+    type
+    txid
+    txStatus
+    senderAddress
+    senderQAddress
+    entityId
+    optionIdx
+    token
+    amount
+    gasUsed
+    blockNum
+    blockTime
+    createTime
+    createBlockNum
+  `,
+};
+
+const MUTATIONS = {
+  createTopic: `
+    CreateTopic(
+      $version: Int!, 
+      $senderAddress: String!,
+      $name: String!,
+      $options: [String!]!,
+      $resultSetterAddress: String!,
+      $bettingStartTime: String!,
+      $bettingEndTime: String!,
+      $resultSettingStartTime: String!,
+      $resultSettingEndTime: String!
+    ) {
+      createTopic(
+        version: $version,
+        senderAddress: $senderAddress,
+        name: $name,
+        options: $options,
+        resultSetterAddress: $resultSetterAddress,
+        bettingStartTime: $bettingStartTime,
+        bettingEndTime: $bettingEndTime,
+        resultSettingStartTime: $resultSettingStartTime,
+        resultSettingEndTime: $resultSettingEndTime
+      ) {
+        version
+        senderAddress
+        name
+        options
+        resultSetterAddress
+        bettingStartTime
+        bettingEndTime
+        resultSettingStartTime
+        resultSettingEndTime
+      }
+    }
+  `,
+
+  createOracle: `
+    CreateOracle(
+      $version: Int!,
+      $name: String!,
+      $options: [String!]!,
+      $resultSetterAddress: String!,
+      $bettingStartTime: String!,
+      $bettingEndTime: String!,
+      $resultSettingStartTime: String!,
+      $resultSettingEndTime: String!
+      $senderAddress: String!
+    ) {
+      createOracle(
+        version: $version,
+        name: $name,
+        options: $options,
+        resultSetterAddress: $resultSetterAddress,
+        bettingStartTime: $bettingStartTime,
+        bettingEndTime: $bettingEndTime,
+        resultSettingStartTime: $resultSettingStartTime,
+        resultSettingEndTime: $resultSettingEndTime
+        senderAddress: $senderAddress,
+      ) {
+        version
+        name
+        options
+        resultSetterAddress
+        bettingStartTime
+        bettingEndTime
+        resultSettingStartTime
+        resultSettingEndTime
+        senderAddress
+      }
+    }
+  `,
+
+  createBet: `
+    CreateBet(
+      $version: Int!,
+      $senderAddress: String!,
+      $oracleAddress: String!,
+      $optionIdx: Int!,
+      $amount: String!
+    ) {
+      createBet(
+        version: $version,
+        senderAddress: $senderAddress,
+        oracleAddress: $oracleAddress,
+        optionIdx: $optionIdx,
+        amount: $amount
+      ) {
+        version
+        senderAddress
+        oracleAddress
+        optionIdx
+        amount
+      }
+    }
+  `,
+
+  approve: `
```

Comment:
`approve` wont be in a separate mutation call, but it will be a separate transaction

Reply:
gotcha, removed